Cannot build site in Linux

```
$ make publish DRY_RUN=true
mkdir -p tmp
cp -R *linkerd.io tmp/
for fname in $(grep -rnl 'L5D2_STABLE_VERSION' tmp); do \
        sed -i '' 's/L5D2_STABLE_VERSION/stable-2.1.0/g' $fname; \
done
sed: can't read s/L5D2_STABLE_VERSION/stable-2.1.0/g: No such file or directory
sed: can't read s/L5D2_STABLE_VERSION/stable-2.1.0/g: No such file or directory
sed: can't read s/L5D2_STABLE_VERSION/stable-2.1.0/g: No such file or directory
make: *** [Makefile:85: replace-env-L5D2_STABLE_VERSION] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -88,7 +88,8 @@ replace-env-%: has-env-% tmp-sites
 		)
 
 	for fname in $$(grep -rnl '$*' tmp); do \
-		sed -i '' 's/$*/$($*)/g' $$fname; \
+		sed -i.bak 's/$*/$($*)/g' $$fname; \
+		rm $$fname.bak; \
 	done
 
 .PHONY: has-env-%

